src/Backend/Modules/Pages/Domain/ModuleExtra/ModuleExtraRepository.php

```diff
@@ -78,11 +80,13 @@ public function getWidgetDataByModuleAndActionAndItemId(string $module, string $
             ->createQueryBuilder('me')
             ->select('me.data')
             ->where('me.module = :module')
+            ->andWhere('me.type = :type')
             ->andWhere('me.action = :action')
             ->andWhere('me.data LIKE :id')
             ->setParameters(
                 [
                     'module' => $module,
+                    'type' => (string) ModuleExtraType::widget(),
```

Comment:
Don't cast to string, use DBAL Type and add it as a parameter so you can do the same with blocks